packages/back-end/src/api/sdk-connections/validations.ts

```diff
@@ -158,6 +159,7 @@ export async function validatePostPayload(
     includeDraftExperiments = false,
     includeExperimentNames = false,
     includeRedirectExperiments = false,
+    includeRuleIds = false,
```

Comment:
Should this also enforce the new default-true behavior?
```suggestion
    includeRuleIds = true,
```

Reply:
I think that also affects creating SDK Connections via the API.  I kind of wanted to leave that part alone and 100% backwards compatible.

Reply:
Ah in that case never mind. I assumed we'd want the API to have the new default behavior